project/src/main/control-manager.gd

```diff
@@ -0,0 +1,131 @@
+extends Node
+"""
+Binds the player's input settings to the input map.
+
+Much of this code was adapted from AnJ95's Godot Toolbox Project: https://github.com/AnJ95/godot-toolbox-project
```

Comment:
While AnJ95's code was helpful, maybe push this comment down to the `pretty_string` function -- most of the rest of this code uses your own data structures and is sort of unrecognizable from the source material.

Reply:
I've removed the comments referencing AnJ95's Godot Toolbox Project.